Ladybird/TestLayout/layout_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Why does this exist twice now?

Reply:
sorry, messed it up a bit while moving directories. fixed.